### ISSUE TITLE
Everything in the scene is black

### ISSUE THREAD

**Issue by brucexiaok** [nesrak1/HKWorldEdit2#8]
After I load scene by name or by file, everything in the scene is darkness, It looks like the near/far plane wrong in the scene camera.
![6325AA5B-7CCE-46b4-8AD4-130B37F9E23A](https://user-images.githubusercontent.com/10465523/114818836-0580be00-9def-11eb-91be-e7070b722ccd.png)
Here is my scene view after open level100

**Comment by nesrak1** [comment]
Those are masks, you can select them and hide them by unchecking the checkbox to the left of the gameobject name textbox in the inspector.

**Comment by brucexiaok** [comment]
i dont think so ,because the scene opend looks normally by my friend's computer, i think it maybe some settings problem.
here's the same level (level94)  opend by my/friend computer
![1](https://user-images.githubusercontent.com/10465523/114820054-fac72880-9df0-11eb-875b-528bafbdf613.png)
![2](https://user-images.githubusercontent.com/10465523/114820056-fc90ec00-9df0-11eb-8c23-26f3cbb29306.png)

**Comment by brucexiaok** [comment]
the lighing and detail looks wired from my computer,And I tried another computer, also the same problem

**Comment by nesrak1** [comment]
Well they _are_ masks in level100 which you can hide. There's a few in your lebel94 screenshots as well. The masks are part of the map, that's not a bug with this project.
In terms of the lighting, the editor has a scene lighting option (the light icon on the top left of the scene window) that may be causing it to look dark.
